sdk/eventhub/event-hubs/src/receiver.ts

```diff
@@ -66,24 +74,33 @@ export class EventHubConsumer {
   }
 
   /**
-   * @property Returns the Partition ID to receive events from.
+   * @property The identifier of the Event Hub partition that this consumer is associated with.
+   * Events will be read only from this partition.
    * @readonly
    */
   public get partitionId(): string {
     return this._partitionId;
   }
 
   /**
-   * @property The consumer group from which the handler is receiving
-   * events from.
+   * @property The name of the consumer group that this consumer is associated with.
+   * Events will be read only in the context of this group.
    * @readonly
    */
   get consumerGroup(): string {
     return this._consumerGroup;
   }
 
   /**
-   * @property The ownerLevel value of the underlying consumer, if present.
+   * @property The priority to associated with an exclusive consumer; for a non-exclusive consumer, this value will be null or undefined.
```

Comment:
Same comment as for `EventHubProducerOptions` around the word "priority" instead of "ownerLevel"